### ISSUE TITLE
What about a multiple return value? 

### ISSUE THREAD

**Issue by rudiedirkx** [rudiedirkx/behat2#2]
Let's say we have a step:

```
Given there is a user "Fred" in organization "Fred's Org" 
```

Which is the return value? The user or the organization? What if I want to save both, one in "USER" and one in "ORG"? 
- How to make the step return 2 values? 
- How to differentiate between them in the storage step?

**Comment by rudiedirkx** [comment]
Maybe a step should always return an array, usually of 1:

```
return [$org->id, $user->id];
```

And the storage step then must always assign an array, usually 1:

```
And store those values in "ORG, USER" 
```

**Comment by rudiedirkx** [comment]
If a step returns 3 values, and you care only about the 2nd:

```
And store those values in ",UID," 
```

Like `list()` does.

**Comment by rudiedirkx** [comment]
Fixed in c442982
